docs/UserGuide.md

```diff
@@ -47,140 +44,200 @@ Format: `help`
+> * The order of the keywords does not matter. e.g. `Meeting Project` will match `Project Meeting`
+> * Search can based on task name, date or tags.
+> * Only full words will be matched e.g. `Project` will not match `Projects`
+> * Tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `Project` will match `Project Meeting`
 
 Examples:
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find Meeting`<br>
+  Returns `Project Meeting`
+* `find 23/03`<br>
+  Returns Any task due in `23/03`.
 
-### 2.6. Deleting a person : `delete`
+### 2.7. Deleting a person : `delete`
 
-Deletes the specified person from the address book. Irreversible.<br>
+Deletes the specified task from the task list. <br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. <br>
+> Deletes the task at the specified `INDEX`. <br>
 > The index refers to the index number shown in the most recent listing.<br>
 > The index **must be a positive integer** 1, 2, 3, ...
 
 Examples:
 
 * `list`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
+  Deletes the 2nd task.
+* `find Project`<br>
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the 1st task from the results of the `find` command.
 
-### 2.7. Select a person : `select`
+### 2.8. Select a task : `select`
 
-Selects the person identified by the index number used in the last person listing.<br>
+Selects the task identified by the index number used in the last task listing.<br>
 Format: `select INDEX`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
+> Selects the task and display all details at the specified `INDEX`.<br>
 > The index refers to the index number shown in the most recent listing.<br>
 > The index **must be a positive integer** 1, 2, 3, ...
 
 Examples:
 
 * `list`<br>
   `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
+  Selects the `2nd task`.
+* `find Project` <br>
   `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  Selects the `1st task` from the results returned from the `find` command.
+  
+### 2.9. Sorting the list of tasks: `sort`
 
-### 2.8. Clearing all entries : `clear`
+Sort all the tasks by due dates, closest to furthest from the current date. <br>
+Format : `sort`
 
-Clears all entries from the address book.<br>
+### 2.10. Undo previous command: `undo` 
+
+Undo previous add/ delete command. <br>
+Format: `undo`
+
+### 2.11. Clearing all entries : `clear`
+
+Clears all entries from the current task list.<br>
 Format: `clear`
 
-### 2.9. Exiting the program : `exit`
+### 2.12. Setting Priority: `priority`
+
+Set Priority for a task with 1 being the most important and 3 being the least important. <br>
+Format: `priority t/TASK p/INDEX`
```

Comment:
usability issue: may be `INDEX` is not the right one to indicate the priority. this is because, you use `INDEX` elsewhere in the user guide to indicate the task index.